app/src/test/kotlin/org/jellyfin/androidtv/data/repository/NetworkStatusRepositoryTest.kt

```diff
@@ -0,0 +1,27 @@
+package org.jellyfin.androidtv.data.repository
+
+import io.kotest.core.spec.style.ShouldSpec
+import io.kotest.matchers.shouldBe
+import io.mockk.mockk
+
+class NetworkStatusImplTest : ShouldSpec({
```

Comment:
Preferably use "FunSpec", although if there's a good reason to use "ShouldSpec" that _should_ be fine.